### PR TITLE
Release optimizations

### PR DESCRIPTION
**Pull request by tburry**
These are some small optimizations that are hopefully okay to do this close to a release.

- There is an every page component that was querying without an available cached endpoint.
- Models were not shared which would cause duplicates to query for their schema info multiple times.

### CODE REVIEW

File: bootstrap.php [hunk 1]
```diff
@@ -318,6 +318,9 @@
     ->rule('Gdn_Model')
     ->setShared(true)
 
+    ->rule(Model::class)
+    ->setShared(true)
```
Comment (charrondev) [inline bootstrap.php:322]:
```suggestion
    ->rule(Model::class)
    ->setInherit(true)
    ->setShared(true)
```

Does this actually do anything without a `setInherit`? All of our model instances are subclasses, and we never pull them out of the container as just `Model`.

Comment (tburry) [reply bootstrap.php:322]: I had checked the code and I think inherit is the default:

```
// Don't add the rule if it doesn't say to inherit.
if (!isset($this->rules[$class]) || (isset($this->rules[$class]['inherit']) && !$this->rules[$class]['inherit'])) {
    continue;
}
```

Comment (tburry) [reply bootstrap.php:323]: I'll confirm with a debug session too. One sec...

Comment (tburry) [reply bootstrap.php:323]: Okay. I confirmed that inherit works, but there was a bug with the rule. I didn't import the namespace.

**Pull request merged**